web/src/components/studies/Studies.tsx

```diff
@@ -1,18 +1,23 @@
 import { useAuth } from "@/hooks/useAuth";
 import LoginFallback from "@/components/ui/LoginFallback";
-import Title from "@/components/ui/Title";
+import ApprovedResearcherView from "./ApprovedResearcherView";
 
 export default function Studies() {
-  const { authInProgress, isAuthed } = useAuth();
-
+  const { authInProgress, isAuthed, userData } = useAuth();
   if (authInProgress) return null;
 
   if (!isAuthed) return <LoginFallback />;
 
+  const isApprovedResearcher = userData?.roles.includes("approved-researcher");
+  const isAdmin = userData?.roles.includes("admin");
+
+  const username = userData!.username;
+
   return (
     <>
-      <Title text={"Studies"} />
-      <p>This page is being built. Please check back soon for updates!</p>
+      {!isAdmin && !isApprovedResearcher && <p>This page is being built. Please check back soon for updates!</p>}
+
```

Comment:
it's not obvious to me that I need to become an approved researcher before I can create a study – I wonder if we should have a "No studies" placeholder and the create button disabled with a reason+prompt to become an approved researcher on hover?